lib/bindings/langs/android/lib/src/main/kotlin/breez_sdk_liquid_notification/job/SwapUpdated.kt

```diff
@@ -121,6 +145,28 @@ class SwapUpdatedJob(
         }
     }
 
+    private fun notifyPaymentWaitingFeeAcceptance(payment: Payment) {
+        if (!this.notified) {
+            logger.log(TAG, "Payment ${payment.txId} requires fee acceptance", "INFO")
```

Comment:
nit: there won't be a txId at this point

Reply:
Thanks for catching that! Fixed in [0c21c7e](https://github.com/breez/breez-sdk-liquid/pull/634/commits/0c21c7e7cda778277668a9ad3dbf2a317dae65b3)